java/org.travlyn.server/src/main/java/org/travlyn/server/externalapi/access/OpenRouteRequest.java

```diff
@@ -42,15 +73,16 @@ public OpenRouteRequest(double latitude, double longitude, CityEntity city) {
         for (int row = -1; row <= 1; row++) {
             for (int column = -1; column <= 1; column++) {
                 APIRequest request = new APIRequest(BASE_URL, new HashSet<>(), this.genPostBody(
-                        BigDecimal.valueOf(latitude + (row * 0.04)).setScale(2, RoundingMode.FLOOR).doubleValue(),
-                        BigDecimal.valueOf(longitude + (column * 0.04)).setScale(2, RoundingMode.FLOOR).doubleValue(),
-                        BigDecimal.valueOf(latitude + ((row + 1) * 0.04)).setScale(2, RoundingMode.FLOOR).doubleValue(),
-                        BigDecimal.valueOf(longitude + ((column + 1) * 0.04)).setScale(2, RoundingMode.FLOOR).doubleValue(),
-                        latitude + (row * 0.04),
-                        longitude + (column * 0.04)), header);
+                        BigDecimal.valueOf(latitude + (row * 0.02)).setScale(2, RoundingMode.FLOOR).doubleValue(),
+                        BigDecimal.valueOf(longitude + (column * 0.02)).setScale(2, RoundingMode.FLOOR).doubleValue(),
+                        BigDecimal.valueOf(latitude + ((row + 1) * 0.02)).setScale(2, RoundingMode.FLOOR).doubleValue(),
+                        BigDecimal.valueOf(longitude + ((column + 1) * 0.02)).setScale(2, RoundingMode.FLOOR).doubleValue(),
+                        latitude + (row * 0.02),
+                        longitude + (column * 0.02)), header);
                 String apiResult;
                 try {
                     apiResult = request.performAPICallPOST();
+                    System.out.println(apiResult);
```

Comment:
Use logger to log this or remove this line as this is only for debugging purpose (`logger.debug(apiResult)`)